client/js/components/EnvironmentVariables/EnvironmentVariables.less

```diff
@@ -0,0 +1,22 @@
+// EnvironmentVariables.less
+@import "../../../less/variables";
+
+.environmentVariables {
+	overflow-y: scroll !important;
+}
+
+.environmentVariables-Notice--error {
+	background-color: @color-error-bg !important;
```

Comment:
No worries - I can update it. I completely forgot about that and face-palmed as soon as I saw your comment :smile_cat: